webots_ros2_driver/src/Driver.cpp

```diff
@@ -38,6 +40,24 @@ int main(int argc, char **argv) {
 
   std::shared_ptr<webots_ros2_driver::WebotsNode> node = std::make_shared<webots_ros2_driver::WebotsNode>(robotName);
   node->init();
+
+  // The parent process must be ros2 run. Declaring from launch file is deprecated. Remove with 2024.0.0.
+  const int BUFFER_SIZE = 4096;
+  const pid_t parentPid = getppid();
+  char processFile[BUFFER_SIZE];
+  sprintf(processFile, "/proc/%d/cmdline", parentPid);
+  int fd = open(processFile, O_RDONLY);
```

Comment:
We should check the return value and exit with an error message in case the file cannot be open.